tests/unit/amo/components/TestSearchResults.js

```diff
@@ -48,7 +42,6 @@ describe('<SearchResults />', () => {
     const root = renderResults({ filters: {} });
     const addonsCard = findRenderedComponentWithType(root, AddonsCard);
 
-    expect(root.message.textContent).toContain('enter a search term');
```

Comment:
I think you can change this test's spec name... this component won't render an error anymore (that's good!) but just renders an empty result set. Maybe changing it to "renders no results when there are no search filters" would work?